test_unstructured/partition/test_auto.py

```diff
@@ -1052,18 +1052,26 @@ def test_add_chunking_strategy_on_partition_auto_respects_max_chars():
 def test_add_chunking_strategy_chars_on_partition_auto_adds_is_continuation():
     filename = "example-docs/example-10k-1p.html"
 
-    # default chunk size in chars is 200
-    partitioned_table_elements_200_chars = [
+    table_elements = [
+        e
+        for e in partition(
+            filename,
+        )
+        if isinstance(e, Table)
```

Comment:
could `e for e in partition(filename) if isinstance(e, Table)` all fit on one line?

Reply:
yup looks that way!